variables.tf

```
@@ -38,15 +38,16 @@ variable "ssh_public_key" {
 }
 
 variable "docker_version" {
-  default = "18.06"
+  default = "19.03"
 }
 
 variable "kubernetes_version" {
-  default = "1.12.2"
+  default = "1.15.4"
```

We should use here latest kubernetes `1.15.15` version

```suggestion
  default = "1.15.5"
```